docs/adguard-vpn-for-mac/solving-problems/logs.md

```diff
@@ -15,9 +15,9 @@ By default, AdGuard VPN for Mac uses the standard logging level, that is, the ba
 
 3. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred. If you can't reproduce the problem, specify as accurately as possible when it last occurred.
 
-4. There is a check mark next to the **Attach technical logs**, which means that when you send a report, you also send logs.
+4. There is a check mark next to the **Send detailed system info**, which means that when you send a report, you also send logs.
```

Comment:
<img width="912" alt="image" src="https://user-images.githubusercontent.com/92315747/194079718-31a76c0f-d6c0-4fc4-809a-f503d3d4e63b.png">

As you can see the current wording is correct

Reply:
@gork7777 I wrote down your suggestions and will pass them on to our content team. But in any case, wordings should be changed in the apps first, and then in the Knowledge Base.